keel/Core/helpers.py

```diff
@@ -0,0 +1,51 @@
+import uuid
+import os
+import boto3
+
+from urllib.parse import urlparse
+
+from config.settings.base import (AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, 
+                                    AWS_S3_REGION, AWS_STORAGE_BUCKET_NAME)
+
+def get_s3_confing():
+    s3_config =  boto3.resource('s3',
+                             region_name=AWS_S3_REGION,
+                             aws_access_key_id=AWS_ACCESS_KEY_ID,
+                             aws_secret_access_key=AWS_SECRET_ACCESS_KEY
+                            )                           
+    return s3_config
+
+def generate_unique_id(prefix):
+    return prefix + str(uuid.uuid4())
+
+
+def upload_file_to_s3(file):
```

Comment:
Do we still need tjhis....S3botostorage is doing the same for us?
once we have defined media storage on S3.....then every file field will automaticallly put evrything to s3 by default ...